### PR TITLE
Milestone: created the toolbar and headline of mobile version for  a portfolio

### PR DESCRIPTION
**Pull request by Roseokpe**
Created toolbar
Added headline part of the project

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+        </div>
+        <nav>
+          <div>
+            <a class="nav-button" href="#"
+              ><img src="images/home-icon.png" alt=""
+            /></a>
+          </div>
+        </nav>
+      </div>
+    </div>
```
Comment (uwadonat) [inline index.html]: - [ ] The container of this section should be a header tag

File: index.html [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+
+    <!--    HEADLINE      -->
+    <div class="head-container">
+      <div class="headline">
```
Comment (uwadonat) [inline index.html]: - [ ] The container of the section should be a section tag

File: index.html [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+        <div class="primary-text">
+          <p>Rose Okpe</p>
```
Comment (uwadonat) [inline index.html]: - [ ] Let's use `h1` tag here to meet project specification

File: index.html [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+            <h4>Skills</h4>
+            <div class="list-skills">
+              <p>Codekit</p>
+              <p>Github</p>
+              <p>Codepen</p>
+              <p>Gitlab</p>
+              <p>Terminal</p>
+            </div>
+          </div>
+        </div>
```
Comment (uwadonat) [inline index.html]: - [ ] Let use `ul` tag and `li` to group this buttons.Check this [link ](https://www.w3schools.com/tags/tag_ul.asp)for `ul` tag

**Review by uwadonat — CHANGES_REQUESTED** [review]
Hi **@[Roseokpe](https://github.com/Roseokpe),** 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>
almost

## Highlights
- Your commit messages are very descriptive ✔️
- No linter error ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 4 above)
Comment (Roseokpe) [reply index.html]: @uwadonat thank you for the review but the last request not clear to me. You want me to use ul tag and li to the listed skills, frameworks and Languages but that will give these items bullet sign which are not in the design.

File: index.html [hunk 5]
```diff
@@ -0,0 +1,111 @@
...
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <div class="header-container">
```
Comment (SajjadAhmad14) [inline index.html]: Please use the `header` tag as the container of the toolbar section as it's requested by the previous code reviewer. Kindly follow semantic HTML. You may read more about HTML CSS best practices [here](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md).

File: index.html [hunk 6]
```diff
@@ -0,0 +1,111 @@
...
+            <a class="nav-button" href="#"
+              ><img src="images/home-icon.png" alt=""
+            /></a>
+          </div>
+        </nav>
+      </div>
+    </div>
+
+    <!--    section     -->
+    <div class="section-container">
```
Comment (SajjadAhmad14) [inline index.html]: Please use the `section` tag as the container of the headline section as it's requested by the previous code reviewer. Kindly follow semantic HTML. You may read more about HTML CSS best practices [here](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md).

File: index.html [hunk 7]
```diff
@@ -0,0 +1,111 @@
...
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <div class="header-container">
+      <div class="toolbar">
+        <div class="logo">
+          <p>Rose Okpe</p>
```
Comment (SajjadAhmad14) [inline index.html]: [ OPTIONAL ] Please make the logo a link using `a` tag.

File: index.html [hunk 8]
```diff
@@ -0,0 +1,111 @@
...
+            <a href=""
+              ><img class="Twitter-icon" src="images/Icon-Twitter.png" alt=""
+            /></a>
+            <a href=""
+              ><img class="Medium-icon" src="images/Medium-icon.png" alt=""
+            /></a>
+          </div>
+          <div class="element-right">
+            <a href=""><img src="images/Group-55.png" alt="" /></a>
+            <p>Work</p>
```
Comment (SajjadAhmad14) [inline index.html]: [ OPTIONAL ] The list of icons should use a `ul` tag to group them, and `li` tag per icon; each icon should have a link `a` tag.

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
## Required Changes ♻️
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hi Rose,
### Great work on making the changes requested by a previous reviewer 👏 
### You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet.
## To Improve 🙏

- [ ] Please check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
#### Please remember to tag me in your question so I can receive the notification.
**_Note: Please do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html [hunk 9]
```diff
@@ -0,0 +1,102 @@
...
+          <a>Rose Okpe</a>
+        </div>
+        <nav>
+          <div>
+            <a class="nav-button" href="#"><img src="images/home-icon.png" alt="#">
+            </a>
+          </div>
+        </nav>
+      </div>
+    </section>
```
Comment (ShadyShawkat) [inline index.html]: - [ ] Kindly replace this `section` tag with a `header` tag, for semantic purposes.

File: index.html [hunk 10]
```diff
@@ -0,0 +1,102 @@
...
+        <hr />
+        <div class="support-item">
+          <div class="social-media">
+            <ul>
+              <li><img class="git-icon" src="images/git-icon.png" alt=" "></li>
+              <li><img class="linkedin-icon" src="images/Linkedin-icon.png" alt=""></li>
+              <li><img class="angelist-icon" src="images/Icon-Angelist.png" alt=""></li>
+              <li><img class="Twitter-icon" src="images/Icon-Twitter.png" alt=""></li>
+              <li><img class="Medium-icon" src="images/Medium-icon.png" alt=""></li>
+            </ul>
```
Comment (ShadyShawkat) [inline index.html]: - [ ] Please wrap the social media links with an `a` tag each. You can use something like the following:
```
<ul>
   <li>
      <a>
         <img> first image
      </a>
   </li>
   <li>
      <a>
         <img> second image
      </a>
   </li>
   ...
</ul>
```

**Review by ShadyShawkat — CHANGES_REQUESTED** [review]
## Hi @Roseokpe, 

Good job so far! 👍🏻 
There are some issues that you still need to work on to go to the next project but you are almost there!

Image

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by AnselemOdims — APPROVED** [review]
### Hi  @Roseokpe 🙋‍♀️, 
Good job in making the required changes 👏 

# STATUS APPROVED ✔
Your project is complete!  There is nothing else to say other than... it's time to merge it 🔗 🚢 
Congratulations! 🎉 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers 🥂  and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Use the tag @AnselemOdims for a quicker reply


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**